### PR TITLE
Added implementation of changes required from FE team

### PR DESCRIPTION
**Pull request by Himan10**

### CODE REVIEW

File: apps/jobs/views.py [hunk 1]
```diff
@@ -427,6 +441,49 @@ def update(self, request, *args, **kwargs):
                 UserSerializer(user_data).data, status.HTTP_200_OK
             )
 
+    @action(detail=False, methods=['get'])
+    def myProfile(self, request):
```
Comment (hims1911) [inline apps/jobs/views.py:445]: change the function name.

File: apps/jobs/constants/values.py [hunk 2]
```diff
@@ -11,6 +11,18 @@
...
+HIRING_STATUS = (
+    ("hiring", "HIRING"),
+    ("open to work", "OPEN TO WORK")
+)
+
+JOB_TYPE = (
+    ("full time", "FULL TIME"),
+    ("part time", "PART TIME"),
+    ("contract", "CONTRACT"),
+    ("internship", "INTERNSHIP")
```
Comment (hims1911) [inline apps/jobs/constants/values.py:23]: I think you can to_lower or to_upper function that will do the work.

File: apps/jobs/models.py [hunk 3]
```diff
@@ -109,6 +111,7 @@ class Meta:
     age = models.PositiveIntegerField(default=None, null=True)
     education = models.TextField(max_length=500, default=None, null=True)
     professional_skills = models.TextField(max_length=500, default=None, null=True)
+    hiring_status = models.CharField(max_length=15, choices=HIRING_STATUS, default="N/A", null=True)
```
Comment (hims1911) [inline apps/jobs/models.py:114]: Default type should not be "NA", It should be pending or "not applied yet".

File: apps/jobs/views.py (hunk 1 above)
Comment (hims1911) [reply apps/jobs/views.py:445]: get_profile_details

**Review by hims1911 — COMMENTED** [review]
provided the comment please have a look

File: apps/jobs/views.py (hunk 1 above)
Comment (Himan10) [reply apps/jobs/views.py]: renamed it to "get_profile_details"

File: apps/jobs/models.py (hunk 3 above)
Comment (Himan10) [reply apps/jobs/models.py]: Replaced it with "Not Applied Yet"

**Pull request merged**